package.json

```diff
@@ -48,5 +58,7 @@
     "lru-cache.js",
     "logo"
   ],
-  "engines": { "node": "0.10.x" }
+  "engines": {
+    "node": ">=0.10.0"
```

Comment:
I mean, for instance, node v5.4.1, which doesn't seem to give acceptable results at least on my machine. The badge width computation is wrong.